ext/RMagick/rmenum.c

```diff
@@ -835,7 +829,6 @@ GravityType_name(GravityType type)
 {
     switch(type)
     {
-        ENUM_TO_NAME(ForgetGravity)
```

Comment:
@dlemstra why was `ForgetGravity` removed here?

Reply:
It has the same value as undefined and that would result in a separate case statement.

Reply:
Ah, I see. Thanks for the explanation!